npm/README.md

```diff
@@ -1,5 +1,7 @@
 # 🤠 wrangler
 
+**Note:** If you're looking for the latest version of the `wrangler` package, visit https://github.com/cloudflare/wrangler2
```

Comment:
Maybe for this and the main README make this whole thing bold? Otherwise seems fine to me

Reply:
There will be additional messaging added during the Deprecation efforts. This was just something missed previously, it was added to local README but not the published one for some reason.